app-frontend/src/app/components/common/search/search.html

```diff
@@ -0,0 +1,15 @@
+<ng-submit ng-submit="$ctrl.onSearchAction()">
+  <div class="form-group search-form">
+    <input id="search-input"
+           class="form-control"
+           placeholder="{{$ctrl.placeholder || 'Search'}}"
```

Comment:
use ng-attr-placeholder for IE support. IE doesn't like {{}} in non-angular properties

Reply:
:+1: